docs/Operators.md

```diff
@@ -13081,7 +13083,13 @@ for test_name, shape in test_cases.items():
         outputs=['reshaped'],
     )
 
-    reshaped = np.reshape(data, shape)
+    # replace zeros with corresponding dim size
+    # we need to do this because np.reshape doesn't support 0
+    new_shape = np.copy(shape)
+    zero_index = np.where(shape == 0)
+    new_shape[zero_index] = np.array(original_shape)[zero_index]
+
+    reshaped = np.reshape(data, new_shape)
```

Comment:
It means that we should have a reference implementation of `ONNX Reshape`. I am thinking something like
```
reshaped = onnx_reshape_impl(…)
```

Reply:
Sure, I'll  do that 👍